### ISSUE TITLE
Plugin does not work with latest Cordova CLI 6.1.1

### ISSUE THREAD

**Issue by elliotstokes** [intercom/intercom-cordova#48]
Please use the following template to submit your issue. Following this template will allow us to quickly investigate and help you with your issue. Please be aware that issues which do not conform to this template may be closed.

For feature requests please contact us at team@intercom.io

## Version info
  - Intercom for Cordova/PhoneGap version: IOS 1.1.4
  - iOS/Android OS version: 9.3.1
  - iOS/Android Device: All tested

## Expected behavior

Plugin should call init and function

## Actual behavior

Plugin does not register and does not initialize and does not allow conversation dispaly etc.
## Steps to reproduce
1. Install Cordova 6.1.1
2. Cordova emulate ios
3. Plugin does not function
4. npm install cordova@5.x
5. Cordova emulate ios
6. plugin works as expected

## Logs

```
 [CDVCommandQueue executePending] [Line 159] FAILED pluginJSON =["INVALID","Intercom","registerIdentifiedUser",[{"userId":"<USERID>"}]]
```

and

```
ERROR: Plugin 'Intercom' not found, or is not a CDVPlugin. Check your plugin mapping in config.xml.
[CDVCommandQueue executePending] [Line 159] FAILED pluginJSON = ["INVALID","Intercom","displayConversationsList",[]]
```

**Comment by spanning** [comment]
Cordova 6.1.1 doesn't read plugins from config.xml. I got it to work by hardcoding the Intercom plugin entry into ios.json like this:

```javascript
                        {
                            "xml": "<feature name=\"Intercom\"><param name=\"ios-package\" value=\"IntercomBridge\" /><param name=\"onload\" value=\"true\" /></feature>",
                            "count": 1
                        }
```

**Comment by kristfal** [comment]
Confirming this issue. @spanning, I'm unable replicate your fix. Could you elaborate?

**Comment by jtreanor** [comment]
Thanks for reporting this issue. It has been fixed in version [1.1.5](https://github.com/intercom/intercom-cordova/releases/tag/1.1.5).
